contracts/src/v0.8/vendor/BufferChainlink.sol

```diff
@@ -309,18 +311,20 @@ library BufferChainlink {
       resize(buf, (len + off) * 2);
     }
 
-    uint mask = 256 ** len - 1;
-    // Right-align data
-    data = data >> (8 * (32 - len));
-    assembly {
-      // Memory address of the buffer data
-      let bufptr := mload(buf)
-      // Address = buffer address + sizeof(buffer length) + off + len
-      let dest := add(add(bufptr, off), len)
-      mstore(dest, or(and(mload(dest), not(mask)), data))
-      // Update buffer length if we extended it
-      if gt(add(off, len), mload(bufptr)) {
-        mstore(bufptr, add(off, len))
+    unchecked {
```

Comment:
Updated from the v0.8 [Buffer.sol](https://github.com/ensdomains/buffer/blob/master/contracts/Buffer.sol), as it relies on over/underflow.

Reply:
[Same here](https://github.com/smartcontractkit/chainlink/pull/4960#discussion_r709055906)